index.html

```diff
@@ -46,69 +52,70 @@ <h1>I'm Benjamin Glad to see you!</h1>
           <img src="photo/logos/Snapshoot-Portfolio.png" alt="">
         </div>
 
-      <div class="hero">
+        <div class="destop-tonic-img">
+          <img src="photo/navs/Snapshoot-Portfolio.jpg" alt="">
+        </div>
+
+        <div class="hero">
           <div class="tonic">
-              <h1>Tonic</h1>
+            <h1>Tonic</h1>
+              <div class="canopy-wrapper">
+                <p>CANOPY</p>
+                  <ul>
+                   <li>Back End Dev</li>
+                     <li>2015</li>
+                  </ul>
+              </div>
```

Comment:
- Please use the proper font size as in the Figma design for your project section.
- Please use the proper styling for your button`See project` and also the technology use `html , CSS, javascript`
- Please remove the horizontal scroll 
Tip: Properly inspect the Figma design properties and apply them to your design so it looks similar to the Figma design.

##### Your design

![Screenshot from 2022-07-16 09-32-56](https://user-images.githubusercontent.com/58771507/179347323-a4680716-25a5-4476-95f7-93e1c537dcc8.png)

##### Figma design

![Screenshot from 2022-07-16 09-33-27](https://user-images.githubusercontent.com/58771507/179347341-82fa1f9f-df4a-4d8b-a043-d140b7312225.png)